util/plancodec/codec_test.go

```diff
@@ -14,45 +14,45 @@
 package plancodec
 
 import (
-	. "github.com/pingcap/check"
 	"github.com/pingcap/tidb/kv"
+	"github.com/stretchr/testify/require"
+	"testing"
 )
 
-var _ = Suite(&testPlanCodecSuite{})
-
-type testPlanCodecSuite struct{}
-
 type encodeTaskTypeCase struct {
 	IsRoot     bool
 	StoreType  kv.StoreType
 	EncodedStr string
 	DecodedStr string
 }
 
-func (s *testPlanCodecSuite) TestEncodeTaskType(c *C) {
+func TestEncodeTaskType(t *testing.T) {
+	t.Parallel()
 	cases := []encodeTaskTypeCase{
 		{true, kv.UnSpecified, "0", "root"},
 		{false, kv.TiKV, "1_0", "cop[tikv]"},
 		{false, kv.TiFlash, "1_1", "cop[tiflash]"},
 		{false, kv.TiDB, "1_2", "cop[tidb]"},
 	}
 	for _, cas := range cases {
-		c.Assert(EncodeTaskType(cas.IsRoot, cas.StoreType), Equals, cas.EncodedStr)
+		require.Equal(t, EncodeTaskType(cas.IsRoot, cas.StoreType), cas.EncodedStr)
```

Comment:
```suggestion
		require.Equal(t, cas.EncodedStr, EncodeTaskType(cas.IsRoot, cas.StoreType))
```

Please be aware different order of position for `expected` and `actual` args.

Reply:
ditto others.

Reply:
@tisonkun Thx, DONE!